blockchain/chain/src/store/chain_store.rs

```diff
@@ -76,49 +76,25 @@ where
 
     /// Writes genesis to blockstore
     pub fn set_genesis(&mut self, header: BlockHeader) -> Result<(), Error> {
```

Comment:
```suggestion
    pub fn set_genesis(&self, header: BlockHeader) -> Result<(), Error> {
```
This doesn't really matter but didn't need mutable ref (maybe we were storing the genesis tipset on the chain store before?)